scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -0,0 +1,540 @@
+
+{% macro omim_phenotypes(variant) %}
+  <div class="card mt-3 panel-default">
+      <div class="panel-heading">OMIM</div>
+      <div class="table-responsive fixed-panel">
+        <table class="table table-bordered table-condensed table-sm">
+          <thead>
+            <tr>
+              <th>Gene</th>
+              <th>Phenotype</th>
+              <th>Inheritance model</th>
+            </tr>
+          </thead>
+          <tbody>
+            {% for gene in variant.genes %}
+              {% if gene.common %}
+                {% for disease_term in gene.common.phenotypes %}
+                  <tr>
+                    <td>
+                        <a href="http://omim.org/entry/{{ gene.common.omim_id }}" target="_blank">
+                          {{ gene.common.hgnc_symbol }}
+                        </a>
+                    </td>
+                    <td>
+                      <a href="http://omim.org/entry/{{ disease_term.mim_number|replace('OMIM:', '') }}" target="_blank">
+                        {{ disease_term.description }}
+                      </a>
+                    </td>
+                    <td>{{ disease_term.inheritance_models|join(', ') }}</td>
+                  </tr>
+                {% endfor %}
+              {% endif %}
+            {% endfor %}
+          </tbody>
+        </table>
+      </div>
+  </div>
+{% endmacro %}
+
+{% macro gtcall_panel(variant) %}
+  <div class="card panel-default">
+    <div class="panel-heading">
+      GT call
+      {% if variant.is_par %}
+        <span class="badge badge-info">PAR</span>
+      {% endif %}
+    </div>
+    <div class="card-body">
+    <div class="table-responsive">
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th rowspan="2">Sample</th>
+            <th rowspan="2">Genotype (GT)</th>
+            <th colspan="2" title="Unfiltered count of reads that support a given allele.">Allele depth (AD)</th>
+            <th rowspan="2" colspan="1" title="Phred-scaled confidence that the true genotype is the one provided in GT (max=99).">Genotype quality (GQ)</th>
```

Comment:
This is not so elegant - or true - right now with SV "GQ"? But, if that other issue is solved quickly, I suppose we could live with it.
![Screenshot 2020-02-05 at 16 36 31](https://user-images.githubusercontent.com/758570/73856532-e0c9a780-4835-11ea-8b8f-c739978e00b5.png)

Reply:
Mm and a bit out of scope for this PR